[2025-02-13T21:34:36.886+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T21:34:36.916+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-06-28T21:17:00+00:00 [queued]>
[2025-02-13T21:34:36.927+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-06-28T21:17:00+00:00 [queued]>
[2025-02-13T21:34:36.928+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-13T21:34:36.945+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2024-06-28 21:17:00+00:00
[2025-02-13T21:34:36.959+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=12569) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-13T21:34:36.960+0000] {standard_task_runner.py:72} INFO - Started process 12575 to run task
[2025-02-13T21:34:36.959+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2024-06-28T21:17:00+00:00', '--job-id', '20023', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpqr5mln45']
[2025-02-13T21:34:36.961+0000] {standard_task_runner.py:105} INFO - Job 20023: Subtask fetch_and_store_crypto_data
[2025-02-13T21:34:37.002+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-06-28T21:17:00+00:00 [running]> on host 7e1fe653b817
[2025-02-13T21:34:37.078+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T21:17:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-28T21:17:00+00:00'
[2025-02-13T21:34:37.079+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-13T21:34:37.140+0000] {logging_mixin.py:190} INFO - Erreur lors de la récupération des données CoinGecko : 429 Client Error: Too Many Requests for url: https://api.coingecko.com/api/v3/simple/price?ids=bitcoin%2Cethereum&vs_currencies=usd
[2025-02-13T21:34:37.141+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-13T21:34:37.148+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-13T21:34:37.149+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2024-06-28T21:17:00+00:00, execution_date=20240628T211700, start_date=20250213T213436, end_date=20250213T213437
[2025-02-13T21:34:37.215+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T21:34:37.222+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
